kimchi/src/circuits/witness/sum_copy_bits_cell.rs

```diff
@@ -0,0 +1,40 @@
+use ark_ff::Field;
+use o1_utils::FieldHelpers;
+
+use super::{variables::Variables, WitnessCell};
+use crate::circuits::polynomial::COLUMNS;
+
+/// Witness cell copied from bits of another witness cell
+pub struct SumCopyBitsCell<F: Field> {
```

Comment:
This is because this is not exactly doing what VariableBitsCell does. It is a VariableBitsCell to which you add the value stored in sum. I guess I can replace this logic within the variable map though, I will give it a try.

Reply:
Worked